docs/enterprise-setup/scaling-airbyte.md

```diff
@@ -0,0 +1,135 @@
+
+To reduce the blast radius of an underprovisioned Airbyte deployment, we recommend placing 'static' workloads (`webapp`, `server`, etc.) on one Kubernetes node group, while placing job-related workloads (connector pods) on a different Kubernetes node group. This ensures that UI or API availability is unlikely to be impacted by the number of concurrent syncs.
+
+<details>
+<summary>Configure Airbyte Self-Managed Enterprise to run in two node groups</summary>
+
+```yaml
+airbyte-bootloader:
+  nodeSelector:
+    type: static
+
+server:
+  nodeSelector:
+    type: static
+
+keycloak:
+  nodeSelector:
+    type: static
+
+keycloak-setup:
+  nodeSelector:
+    type: static
+
+temporal:
+  nodeSelector:
+    type: static
+
+webapp:
+  nodeSelector:
+    type: static
+
+worker:
+  nodeSelector:
+    type: jobs
+
+workload-launcher:
+  nodeSelector:
+    type: static
+  ## Pods spun up by the workload launcher will run in the 'jobs' node group.
+  extraEnvs:
+    - name: JOB_KUBE_NODE_SELECTORS
+      value: type=jobs
+    - name: SPEC_JOB_KUBE_NODE_SELECTORS
+      value: type=jobs
+    - name: CHECK_JOB_KUBE_NODE_SELECTORS
+      value: type=jobs
+    - name: DISCOVER_JOB_KUBE_NODE_SELECTORS
+      value: type=jobs
+
+orchestrator:
+  nodeSelector:
+    type: jobs
+  
+workload-api-server:
+  nodeSelector:
+    type: jobs
+```
+
+</details>
+
+## High Availability
+
+You may wish to implement high availability (HA) to minimize downtime and ensure continuous data integration processes. Please note that this requires provisioning Airbyte on a larger number of Nodes, which may increase your licensing fees. For a typical HA deployment, you will want a VPC with subnets in at least two (and preferably three) availability zones (AZs).
+
+We particularly recommend having multiple instances of `worker` and `server` pods:
+
+```yaml
+worker:
+  replicaCount: 2
+
+server:
+  replicaCount: 2
+```
+
+Furthermore, you may want to implement a primary-replica setup for the database (e.g., PostgreSQL) used by Airbyte. The primary database handles write operations, while replicas handle read operations, ensuring data availability even if the primary fails.
+
+## Disaster Recovery (DR) Regions
+
+For business-critical applications of Airbyte, you may want to configure a Disaster Recovery (DR) cluster for Airbyte. We do not support assisting customers with DR deployments at this time. However, we offer a few high level suggestions:
+1. We strongly recommend configuring an external database, external log storage and external connector secret management.
+2. We strongly recommend that your DR cluster is also an instance of Self-Managed Enterprise, kept at the same version as your prod instance.
+
+## DEBUG Logs
+
+We recommend turning off `DEBUG` logs for any non-testing use of Self-Managed Airbyte. Failing to do while running at-scale syncs may result in the `server` pod being overloaded, preventing most of the deployment for operating as normal.
```

Comment:
```suggestion
We recommend turning off `DEBUG` logs for any non-testing use of Self-Managed Airbyte. Failing to do so while running at-scale syncs may result in the `server` pod being overloaded. This would prevent most of the deployment from operating as normal.
```